presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConnector.java

```diff
@@ -63,7 +63,7 @@
     private final List<PropertyMetadata<?>> tableProperties;
     private final ConnectorAccessControl accessControl;
     private final Set<Procedure> procedures;
-    private final ConnectorPlanOptimizer planOptimizer;
+    private final Set<ConnectorPlanOptimizer> planOptimizers;
```

Comment:
Because `IcebergParquetDereferencePushDown` is implemented in a separate class, this causes the Iceberg connector to have two `ConnectorPlanOptimizer` implementation classes, the other is the `IcebergPlanOptimizer`. We need to pass these two` ConnectorPlanOptimizer` implementation classes to `IcebergPlanOptimizerProvider.getLogicalPlanOptimizers`, which is a return type of `Set<ConnectorPlanOptimizer>`, so I used `Set<ConnectorPlanOptimizer>` to store `planOptimizers` in this piece.

Reply:
Thanks for the clarification!